provider/data/list/fingerprints.csv

```diff
@@ -1,386 +1,1005 @@
-list/unit@2, kea, 1370B, 37d6d34c32e75245
-list/unit@2, kgp, 1376B, e24ecc3b00e68f0d
-list/unit@2, kk, 1356B, -> be
-list/unit@2, km, 1356B, -> be
-list/unit@2, kn, 1367B, 2cb11ac2b35a34ad
-list/unit@2, ko, 1356B, -> be
-list/unit@2, kok, 1364B, -> as
-list/unit@2, ks, 1416B, 65acbf4a1b18ae0d
-list/unit@2, ks-Deva, 1407B, 6fa43478ae4c9ca
-list/unit@2, ku, 1372B, e805f0ffad369693
-list/unit@2, kxv, 1364B, -> as
-list/unit@2, ky, 1364B, -> as
-list/unit@2, lb, 1364B, -> as
-list/unit@2, lo, 1364B, -> as
-list/unit@2, lt, 1362B, bd60ee39ff6c5f41
-list/unit@2, lv, 1372B, fe102e31b56895bb
-list/unit@2, mai, 1407B, 9d7d1b4c5f2fe05e
-list/unit@2, mi, 1364B, -> as
-list/unit@2, mk, 1380B, -> bs-Cyrl
-list/unit@2, ml, 1365B, f986f4b361ed7d92
-list/unit@2, mn, 1356B, -> be
-list/unit@2, mni, 1458B, 43d80f3868995b85
-list/unit@2, mr, 1364B, -> as
-list/unit@2, ms, 1370B, c22e8ca50f17f821
-list/unit@2, mt, 1374B, b5768c3cf6ab2be0
-list/unit@2, my, 1452B, 98a6cfcd9842ecab
-list/unit@2, ne, 1357B, e5817d6b83385ea0
-list/unit@2, nl, 1372B, ad668c721c0bc675
-list/unit@2, nn, 1364B, -> as
-list/unit@2, no, 1372B, a2d30287718b13df
-list/unit@2, or, 1364B, -> as
-list/unit@2, pa, 1364B, -> as
-list/unit@2, pcm, 1364B, -> as
-list/unit@2, pl, 1374B, -> bs
-list/unit@2, ps, 1370B, 4b3e17ded11d6483
-list/unit@2, pt, 1368B, -> it
-list/unit@2, pt-PT, 1374B, fbc347f90f14c92e
-list/unit@2, rm, 1368B, -> it
-list/unit@2, ro, 1371B, a7f7ca3d0ea8b619
-list/unit@2, ru, 1356B, -> be
-list/unit@2, sa, 1425B, f34a7c9f4fc77f86
-list/unit@2, sah, 1428B, a9c1b13116f870d5
-list/unit@2, sc, 1368B, -> it
-list/unit@2, si, 1407B, 98beb0a0883f6fdb
-list/unit@2, sl, 1380B, 648a182016ff2887
-list/unit@2, so, 1371B, a44cf38ba21358a0
-list/unit@2, sq, 1374B, -> pt-PT
-list/unit@2, sr, 1380B, -> bs-Cyrl
-list/unit@2, sr-Latn, 1374B, -> bs
-list/unit@2, su, 1407B, f71496632930aba0
-list/unit@2, sv, 1364B, -> as
-list/unit@2, sw, 1380B, 306477c695c8cea1
-list/unit@2, syr, 1380B, cc45ef7002f5f7f2
-list/unit@2, ta, 1364B, -> as
-list/unit@2, th, 1386B, 3634af6f27539959
-list/unit@2, ti, 1389B, 51e44db4390c0322
-list/unit@2, tk, 1364B, -> as
-list/unit@2, to, 1390B, 647c23b950566c92
-list/unit@2, tr, 1356B, -> be
-list/unit@2, tt, 1404B, c50a7526dc58c855
-list/unit@2, ug, 1389B, 35c139f801b2bcb9
-list/unit@2, uk, 1380B, cb1ace7c262d0af6
-list/unit@2, und, 1368B, 924e2d2be2336604
-list/unit@2, ur, 1409B, b92dbfa5baac3d9c
-list/unit@2, uz, 1356B, -> be
-list/unit@2, vec, 1368B, -> it
-list/unit@2, vi, 1364B, -> as
-list/unit@2, xnr, 1366B, 5f6f58501e522c14
-list/unit@2, yrl, 1384B, fcc6d6177fc86427
-list/unit@2, yue, 1352B, -> ja
-list/unit@2, yue-Hans, 1352B, -> ja
-list/unit@2, zh, 1344B, 16c295fb252e0c3
-list/unit@2, zh-Hant, 1352B, -> ja
-list/unit@2, zu, 1367B, 6202988c3f3e90c9
+list/and@2, <lookup>, 1916B, 355 identifiers
+list/and@2, <total>, 39389B, 127 unique payloads
+list/and@2, af/N, 290B, d2846cc3cf177584
+list/and@2, af/S, 288B, abf92122ab812de
+list/and@2, af/W, 288B, -> af/S
```

Comment:
Discussion: you use `N`, `S`, `W` here; I think Younies or someone was using `l` and `s`, and in datetime we use numbers `1`, `3`, `4` (2 means 2-digit and isn't part of the data marker attribute). Should we be more consistent? (Doesn't block this PR)

Reply:
i'd prefer lowercase here, this feels like cardinal directions

extremely light preference, and very much non blocking